### DISCUSSION TITLE
[Suggestion] Decal Removal Command

### DISCUSSION THREAD

**Discussion by Dwarf-LordPangolin** [tsunamayo/Starship-EVO#3963] [Suggestions Done]
It would be handy to have a command or tool that can either remove all decals from a creation, or at least remove them in a chunk. This would be useful for two reasons.

First, the novox update is _really freakin' awesome_! However, it has left a number of builds covered in so many decals that in a couple instances it's been faster to rebuild the object from scratch than to try to remove them all.
![20210706144356_1](https://user-images.githubusercontent.com/33041786/124837145-e86d0e80-df49-11eb-9032-48a6768762a3.jpg)

This is particularly problematic when they are on faces between two pieces:
![20210705164406_1](https://user-images.githubusercontent.com/33041786/124837178-f6229400-df49-11eb-8a3b-650832b9e3a1.jpg)

Second, there are cases (especially when copy-pasting something goes awry) where the thing pasted needs to be deleted. This is slow enough without a mass delete, but when it has decals on it, this gets a lot slower. This is a particular problem when you only realize it has decals after you've been deleting for a while, since floating decals cannot be removed: they need a surface under them.
![20210707173058_1](https://user-images.githubusercontent.com/33041786/124837429-67624700-df4a-11eb-8531-21fff7fce2bb.jpg)
![20210707173259_1](https://user-images.githubusercontent.com/33041786/124837437-6b8e6480-df4a-11eb-9414-54810d0f1956.jpg)

In summary, a command or tool for mass decal removal would be handy, at least for cleanup after the novox update.

**Comment by ExodistSKY1** [comment]
I have also ran into this problem I think it was something to do with the previous paint transferring over. I have a build with over 23k decals and I would truly love a Delete all decals command for this

**Comment by tsunamayo** [comment]
Hi, I see. I wanted to give a mass delete tool for blocks. I could add an option into F1 that says : Mass Delete Tool on Decals only.
Would this works for yo guys? 
Cheers

**Comment by ExodistSKY1** [comment]
This would be perfect

**Reply by Dwarf-LordPangolin** [reply]
Agreed; that would be ideal!

**Comment by tsunamayo** [comment]
Okay box delete was just added.
